apps/generic_terminal/unix_utilities.py

```diff
@@ -0,0 +1,90 @@
+from talon import Context, Module
+
+from ...code.user_settings import get_list_from_csv
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+tag: terminal
+and tag: user.unix_utilities
+"""
+
+mod.tag(
+    "unix_utilities", desc="tag for enabling unix utility commands in your terminal"
+)
+
+# Do not edit this dictionary. It is just used to initially populate 'settings/unix_utilities.csv'.
```

Comment:
I thought about that and looked around how this sort of stuff is implemented in other places. Since these commands are just the initial content of `settings/unix_utilities.csv`, I figured I should take inspiration from `file_extensions.py` and `vocabulary.py`. These also use a dictionary for the initial content. My rationale is that having only one CSV file makes at less confusing which file you need to edit if you want to customize your utility commands.

Therefore I would be in favour of keeping it the wait currently is. I would be happy to follow the consensus, though.